sgkit/tests/io/vcf/utils.py

```diff
@@ -1,12 +1,103 @@
+from itertools import zip_longest
 from pathlib import Path
 
+import numpy as np
+
+from sgkit.io.vcf.vcf_reader import open_vcf
 from sgkit.typing import PathType
 
 
 def path_for_test(shared_datadir: Path, file: str, is_path: bool = True) -> PathType:
     """Return a test data path whose type is determined by `is_path`.
 
-    If `isPath` is True, return a `Path`, otherwise return a `str`.
+    If `is_path` is True, return a `Path`, otherwise return a `str`.
     """
     path: PathType = shared_datadir / file
     return path if is_path else str(path)
+
+
+def assert_allclose(f1, f2, *, rtol=1e-05, atol=1e-03):
```

Comment:
Is this going to clash with the numpy.testing.assert_allclose though? I can imagine both functions being used in the same file and the ``from module import function`` style makes it hard to know where particular functions come from. How about ``assert_vcfs_close``?

Reply:
I wondered about this - I agree this makes things less ambiguous and clash-prone.